polaris.shopify.com/src/components/MobileNav/MobileNav.module.scss

```diff
@@ -26,23 +20,36 @@ $navIconSize: 24px;
     transition: transform var(--duration-200) var(--ease-in-out);
 
     .CloseButton {
-      position: absolute;
       display: flex;
-      top: calc(var(--header-height) / 2 - $navIconSize / 2);
-      right: calc(-1rem - $navIconSize);
+      margin: 0.85rem;
       background-color: transparent;
       opacity: 0;
       animation: FadeIn var(--duration-200) forwards;
 
       svg {
-        width: 24px;
         fill: var(--surface);
       }
     }
   }
+}
+
+.LinksList {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  height: 100%;
+  width: calc(100vw - 4rem);
+  max-width: 22.5rem;
```

Comment:
This `22.5rem` value makes a few appearances. Extract into a global CSS variable? `--mobile-nav-max-width`